docs/developer/best-practices/performance.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[plugin-performance]]
```

Comment:
Once we've published these docs it would be nice to link to it from the CI Reporter so that a developer can come here from their PR to read how to improve the bundle size